kubernetes/helm/thirdeye/templates/common/configmap.yaml

```diff
@@ -0,0 +1,252 @@
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: thirdeye-pinot-quickstart-datasource-config
+data:
+  persistence.yml: |-
+    databaseConfiguration:
+      url: jdbc:mysql://{{ .Release.Name }}-mysql/thirdeye
+      user: {{ .Values.mysql.mysqlUser }}
+      password: {{ .Values.mysql.mysqlPassword }}
+      driver: com.mysql.jdbc.Driver
+  dashboard.yml: |-
+    authConfig:
+      authEnabled: false
+      authKey: ""
+      ldapUrl: ""
+      domainSuffix: []
+      cacheTTL: 3600
+      cookieTTL: 604800
+    resourceConfig: []
+    rootCause:
+      definitionsPath: rca.yml
+      parallelism: 5
+      formatters:
+        - org.apache.pinot.thirdeye.dashboard.resources.v2.rootcause.AnomalyEventFormatter
+        - org.apache.pinot.thirdeye.dashboard.resources.v2.rootcause.ThirdEyeEventFormatter
+        - org.apache.pinot.thirdeye.dashboard.resources.v2.rootcause.MetricEntityFormatter
+        - org.apache.pinot.thirdeye.dashboard.resources.v2.rootcause.DimensionEntityFormatter
+        - org.apache.pinot.thirdeye.dashboard.resources.v2.rootcause.ServiceEntityFormatter
+        - org.apache.pinot.thirdeye.dashboard.resources.v2.rootcause.DefaultEventEntityFormatter
+    dashboardHost: "http://localhost:1426"
+    failureFromAddress: thirdeye@localhost
+    failureToAddress: user@localhost
+    alerterConfiguration:
+      smtpConfiguration:
+        smtpHost: localhost
+        smtpPort: 25
+    logging:
+      level: INFO
+      loggers:
+        org.apache.pinot: INFO
+
+    server:
+      type: default
+      applicationConnectors:
+        - type: http
+          port: 1426
+      adminConnectors:
+        - type: http
+          port: 1427
+      requestLog:
+        type: external
+    whitelistDatasets: []
+    swagger:
+      resourcePackage: "org.apache.pinot.thirdeye.dashboard.resources,org.apache.pinot.thirdeye.dashboard.resources.v2,org.apache.pinot.thirdeye.anomaly.onboard,org.apache.pinot.thirdeye.detection,org.apache.pinot.thirdeye.detection.yaml"
+
+  detector.yml: |-
+    logging:
+      level: INFO
+      loggers:
+        org.hibernate.engine.internal: WARN
+    server:
+      type: default
+      rootPath: '/api/*'
+      applicationContextPath: /
+      adminContextPath: /admin
+      applicationConnectors:
+        - type: http
+          port: 1867
+      adminConnectors:
+        - type: http
+          port: 1868
+    alert: false
+    autoload: true
+    # autoOnboardConfiguration:
+    #   runFrequency: 10s
+
+    classifier: false
```

Comment:
We'll need separate "detector.yml" files for the **scheduler backend** and regular **worker backends**. Scheduler can be a worker too, but there can only be one scheduler active at a time.

**Scheduler**
alert: true
autoload: true
classifier: true
holidayEventsLoader: true
monitor: true
scheduler: true
worker: true/false
detectionPipeline: true
detectionAlert: true

**Worker**
alert: false
autoload: false
classifier: false
holidayEventsLoader: false
monitor: false
scheduler: false
worker: true
detectionPipeline: false
detectionAlert: false

Reply:
I'm creating a composite worker/scheduler pod for now just to keep things simple. I plan to split these up later on since these configurations hasn't been tested.